model-optimizer/extensions/front/mxnet/gather.py

```diff
@@ -26,10 +26,10 @@ class GatherFrontReplacer(FrontReplacementOp):
 
     def replace_sub_graph(self, graph: Graph, match: dict):
         node = match['op']
-        gather_node = Gather(graph, dict(name=node.id + '/embedding_',
-                                         symbol_dict={'name': node.id + '/embedding_'})).create_node()
-        axis_const = Const(graph, {'value': int64_array(0)}).create_node()
+
+        gather_node = create_op_with_const_inputs(graph, Gather, {2: int64_array(0)}, {'name': node.id + '/embedding_',
```

Comment:
Please, do not use `node.id`. Use regular `node.soft_get('name', node.id)` instead. 

Also, get rid of `symbol_dict` from the code. It should not be used at all.

Reply:
Done.